### DISCUSSION TITLE
Add for jellyfin app and web tv list country metada

### DISCUSSION THREAD

**Discussion by petitbou0169** [jellyfin/jellyfin#5400] [General]
Hello friends, 

this is what we have developed for tv in order to be able to manage channels from several countries for iptv.

Can we integrate it on expo and on the web version ?

I'm attaching 2 screens to give you an idea

[url=https://zupimages.net/viewer.php?id=21/09/3jrw.png][img]https://zupimages.net/up/21/09/3jrw.png[/img][/url]

and

[url=https://zupimages.net/viewer.php?id=21/09/hfor.png][img]https://zupimages.net/up/21/09/hfor.png[/img][/url]

**Comment by crobibero** [comment]
The images you linked are password protected, so I’m not 100% sure what you’re looking for. At the moment to manage multiple playlists we’ve been referring people to use [xTeVe](https://github.com/xteve-project/xTeVe)  as Jellyfin doesn’t have that capability.
